.github/workflows/main.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
       - name: Install
         run: |
           git config --global --add safe.directory /__w/cmip6-downscaling/cmip6-downscaling
-          python -m pip install --no-deps -e .
-          python -m pip install -U intake intake-esm cf-xarray
+          python -m pip install -e .
```

Comment:
@orianac 

https://github.com/carbonplan/cmip6-downscaling/pull/275/commits/926c30c31dd696b54c4ba568f077603332916f59

Seems like the tests fail without that. 

Not super familiar with how the CI works, but maybe the correct way is to update / create a new image? 

Maybe @andersy005, do you have any recs here?

Reply:
btw I think removing --no-deps is fine here and preferable to creating a new image. Otherwise it will only install cmip6-downscaling and ignore the version specifications included in requirements.txt.